test/integration/health_check_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+
+class HealthCheckTest < ActionDispatch::IntegrationTest
+
+  def setup
+    super
+    @headers = { 'Accept': 'application/json' }
+  end
+
+  def test_default_check_passed
+    endpoint = 'default'
+    get("/healthcheck/#{endpoint}.json", headers: @headers)
+    keys = parse_keys(response: response, endpoint: endpoint)
+
+    assert_health_check_keys(keys)
+  end
+
+  def test_db_check_passed
+    endpoint = 'database'
+    get("/healthcheck/#{endpoint}.json", headers: @headers)
+    keys = parse_keys(response: response, endpoint: endpoint)
+
+    assert_health_check_keys(keys)
+  end
+
+  def test_api_check_passed
+    endpoint = 'api'
+    get("/healthcheck/#{endpoint}.json", headers: @headers)
+    keys = parse_keys(response: response, endpoint: endpoint)
+
+    assert_health_check_keys(keys)
+  end
+
+  def test_registry_check_passed
+    endpoint = 'registry'
+    get("/healthcheck/#{endpoint}.json", headers: @headers)
+    keys = parse_keys(response: response, endpoint: endpoint)
+
+    assert_health_check_keys(keys)
+  end
+
+  def test_tara_check_passed
+    endpoint = 'tara'
+    get("/healthcheck/#{endpoint}.json", headers: @headers)
+    keys = parse_keys(response: response, endpoint: endpoint)
+
+    assert_health_check_keys(keys)
+  end
+
+  def test_email_check_passed
+    endpoint = 'email'
+    get("/healthcheck/#{endpoint}.json", headers: @headers)
+    keys = parse_keys(response: response, endpoint: endpoint)
+
+    assert_health_check_keys(keys)
+  end
+
+  def test_sms_check_passed
```

Comment:
What do you think about grouping the statements according to `given/when/then` steps visually?

Reply:
Like this:
 
```
# Given. absent?

# When.
get("/healthcheck/sms.json", headers: @headers)

# Then.
keys = parse_keys(response: response, endpoint: endpoint)
assert_health_check_keys(keys)
```

Reply:
P.S. Comments are just to explain my point. No need to keep them.

Reply:
Currently it's not clear at first sight, where is what.

Reply:
I suppose here we've got given: endpoint.